core/src/apps/cardano/address_id.py

```diff
@@ -0,0 +1,8 @@
+from micropython import const
+
+
+BASE_ADDRESS_KEY_KEY = const(0)
```

Comment:
probably typo -> `KEY` twice

Reply:
It wasn't a typo, more values where possible for the address id:
`BASE_ADDRESS_KEY_KEY = const(0)
BASE_ADDRESS_SCRIPT_KEY = const(1)
BASE_ADDRESS_KEY_SCRIPT = const(2)
BASE_ADDRESS_SCRIPT_SCRIPT = const(3)`
But I've decided to drop the unusued values. They are not planned for the initial Shelley release anyways.